graphiti_core/search/search_utils.py

```
@@ -748,42 +793,47 @@ async def get_relevant_edges(
-    )
+    try:
+        results, _, _ = await driver.execute_query(
+            query,
+            params=query_params,
+            edges=[edge.model_dump() for edge in edges],
+            limit=limit,
+            min_score=min_score,
+            database_=DEFAULT_DATABASE,
+            routing_='r',
+        )
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)